app/experimenter/nimbus-ui/src/lib/experiment.ts

```diff
@@ -0,0 +1,38 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { getExperiment_experimentBySlug } from "../types/getExperiment";
+import { NimbusExperimentStatus } from "../types/globalTypes";
+
+export type StatusCheck = {
+  draft: boolean;
+  review: boolean;
+  accepted: boolean;
+  live: boolean;
+  complete: boolean;
+  locked: boolean;
+  released: boolean;
+};
+
+export function getStatus(
```

Comment:
Might be good to have a comment above this function so one can quickly see what `released` and `locked` means.